README.md

```diff
@@ -1,11 +1,8 @@
 # uv
 
-[![uv](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/uv/main/assets/badge/v0.json)](https://github.com/astral-sh/uv)
```

Comment:
Honestly not sure what this one does in this context, intended for the PyPI README I guess?